app/uk/gov/hmrc/apinotificationqueue/controller/QueueController.scala

```diff
@@ -45,6 +46,8 @@ class QueueController @Inject()(queueService: QueueService, fieldsService: ApiSu
   def save(): Action[AnyContent] = Action.async {
     implicit request => {
       val headers = request.headers
+      val message = headers.get("X-Conversation-ID").fold(s"[conversationId not found] Headers=$headers"){ cid => s"[conversationId=$cid]"}
+      Logger.debug(s"[QueueController]$message Request received")
```

Comment:
I would change it to:
```
Logger.debug(s"Request received - $message")
```

Reply:
If you want to include `[QueueController]` in the log message, you could also write `[QueueController#save()]`, so you know what method has been called

Reply:
@gokyo changed and pushed.